src/resolvers/ReviewResolver.ts

```diff
@@ -0,0 +1,277 @@
+import { Resolver, Query, Mutation, Arg, Ctx, UseMiddleware } from "type-graphql";
+import { ReviewCampaign, ReviewQuestion, Review, ReviewAnswer, DropDown } from "../models/reviews";
+import { CreateReviewCampaignInput, UpdateReviewCampaignInput, AddQuestionToCampaignInput, UpdateQuestionInput, SubmitReviewInput, UpdateReviewInput, CreateReviewCampaignPayload } from "../models/types/reviews";
+import { Events } from "../models/organizers/events/Events";
+import { isAuth } from "../middleware/isAuth";
+
+@Resolver()
+export class ReviewResolver {
+    // Review Campaign Queries
+    @Query(() => [ReviewCampaign])
+    async getReviewCampaigns(@Arg("eventId") eventId: string) {
+        return await ReviewCampaign.find({
+            where: { eventId: { id: eventId } },
+            relations: ["questions", "questions.dropDown", "reviews"]
+        });
+    }
+
+    @Query(() => ReviewCampaign, { nullable: true })
+    async getReviewCampaign(@Arg("id") id: string) {
+        return await ReviewCampaign.findOne({
+            where: { id },
+            relations: ["questions", "questions.dropDown", "reviews", "reviews.answers"]
+        });
+    }
+
+    @Query(() => [ReviewCampaign])
+    async getReviewCampaignsForUser(@Arg("eventId") eventId: string, @Arg("ticketType") ticketType?: string) {
```

Comment:
Before you can send review_campaign you need to verify user is logged in and authenticated. And your sending them the right review, and not sending them the wrong thing